### PR TITLE
Travis CI setup

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Travis CI setup

## Type of change

- CI setup
- 
## Testing steps

Let's try the CI

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  apt:
+    packages:
+    - libsnappy-dev
+# Pycodestyle part
+# needed to work correctly with Python 3 shebang
+env: SKIP_INTERPRETER=true
+install:
+  - pip install pycodestyle
+script:
+  - python run_pycodestyle.py
```
Comment (joselsegura) [inline .travis.yml:17]: Why don't use directly `pycodestile`? In addition, as probably the number of linters will grow, maybe it is better to move pycodestyle to a `requirements-dev.txt` file and just install the dependencies from there.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+  - "3.6"      # current default Python on Travis CI
+  - "3.7"
+  - "3.8"
+  - "nightly"  # nightly build
```
Comment (joselsegura) [inline .travis.yml:6]: Which version is "nightly" right now? A few weeks ago Python 3.10.0 was released

**Review by joselsegura — APPROVED** [review]
Just nitpicks (that can be done in the future when Travis run more linters)

File: .travis.yml (hunk 2 above)
Comment (tisnik) [reply .travis.yml:6]: I thing so

File: .travis.yml (hunk 1 above)
Comment (tisnik) [reply .travis.yml:17]: When I checked it, pycodestyle was unable to find all Python sources spread over multiple directories (which is not one project). Probably it has changed?

File: .travis.yml (hunk 2 above)
Comment (tisnik) [reply .travis.yml:6]: yes 3.10 https://app.travis-ci.com/github/RedHatInsights/insights-results-aggregator-utils/jobs/546635132

File: .travis.yml (hunk 1 above)
Comment (tisnik) [reply .travis.yml:17]: ok now it works, it just does not print total number of errors :)))

Comment (joselsegura) [reply .travis.yml:17]: I usually do `pycodestyle **/*.py` and let bash to find everything, but not sure how would that work on travis.

We had some good tooling around Python code in ccx-data-pipeline repository, using pre-commit and a lot of formatters and linters, but it was like 2 years ago that I don't pay attention to it :-/

Comment (tisnik) [reply .travis.yml:17]: ok will change it

File: .travis.yml (hunk 2 above)
Comment (tisnik) [reply .travis.yml:7]: added 3.9 too

**Pull request merged**